source/reporting/index.html.md.erb

```diff
@@ -87,9 +89,11 @@ have entered when making a payment:
     "city": "London",
     "country": "GB"
   }
-},
+}
 ```
 
+The `card_type` is `null` if we did not recognise which type of card your user made the payment with.
```

Comment:
Given the broad range of reasons why the `"card_type"` might be null, “we were unable to determine” feels more accurate to me that “we did not recognise” but I’m happy to defer to your judgement.

Reply:
If I remember right, I think we started with something along the lines of "we were unable to determine" - but I asked for some tech writer help to tweak it for style, and "we did not recognise" was the consensus. Let's give it a go and tweak it later if users aren't sure what we mean.